foreman/client.py

```diff
@@ -393,20 +394,24 @@ def _get_local_defs(self, strict=True):
             if f_ver.split('.')[:2] == self.version.split('.')[:2]:
                 last_major_match = f_name
             if ver_cmp(f_ver, self.version) < 0:
```

Comment:
I will change it more, because problem can be with line no 389 ("for f_name in sorted(files):")

In [3]: sorted(['1.3.4', '1.20.3', '1.5.0'])
Out[3]: ['1.20.3', '1.3.4', '1.5.0']

In [8]: sorted(['1.3.4', '1.20.3', '1.5.0'], cmp=ver_cmp)
Out[8]: ['1.20.3', '1.3.4', '1.5.0']

In [9]: [1, 20, 3] < [1, 3, 4]  # this is logically correct
Out[9]: False

In [12]: ['1', '20', '3'] < ['1', '3', '4']  # this is logically wrong
Out[12]: True

this case will have problem as well.